app/controllers/movie_controller.rb

```diff
@@ -0,0 +1,40 @@
+class MovieController < ApplicationController
```

Comment:
This controller looks amazing! I am so impressed with all this work. I can't wait to begin refactoring with you using Facades and Services! EPIC JOB!